brian-tome.html

```diff
@@ -74,11 +74,13 @@ <h3 class="text-blue component-header flush-top push-half-bottom">
   </div>
 </section>
 
+<!--
```

Comment:
Please fully remove Instagram. We can easily add it back in the future if needed.

Reply:
@dholloran-ample Fixed!